invitations/app_settings.py

```diff
@@ -98,5 +98,12 @@ def ADMIN_CHANGE_FORM(self):
             "invitations.forms.InvitationAdminChangeForm"
         )
 
+    @property
+    def COMFIRMATION_VIEWNAME(self):
```

Comment:
Can we call it `CONFIRMATION_URL_NAME` since it's the name of the URL pattern we are referring to?

Reply:
I've removed the source repository a long time ago and can't find a way to update this PR. I guess I'll have to create a new branch and new PR

Reply:
Roger that - maybe you can use the .patch as a starting point: https://github.com/jazzband/django-invitations/pull/108.patch